GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+
+## Community Roles and Responsibilities
+
+See <https://github.com/pingcap/community/tree/master/architecture> for details.
+
+## Community Structure
+
+![TiDB Community Architecture](./media/architecture.svg)
+
+The project is comprised of and operated by the following subgroups:
+
+- Project Management Committee (PMC)
+- Organizer Committee
+- Developer Group
+    - Maintainers
+    - Special Interest Groups, SIGs (Tech Leads)
+    - Working Groups, WGs
+- TiDB User Group
+
+### Project Management Committee
+
+The PMC functions as the core management team that oversees the TiDB community. The PMC has additional responsibilities over and above those of Maintainers. These responsibilities ensure the smooth running of the project.
+
+Members of the PMC do not have significant authority over other members of the community, although it is the PMC that votes on new Maintainers or Committers, and makes all major decisions for the future with respect to TiDB, such as project-level governance policies, management of sub-structures, security processes and so on. It also makes decisions when community consensus cannot be reached. In addition, the PMC has access to the project’s private mailing list and its archives.
+
+Membership of the PMC is by invitation from the existing PMC members. A nomination will result in discussion and then a vote by the existing PMC members. PMC membership votes are subject to consensus approval of the current PMC members.
+
+For the first formation of the committee, our maintainers nominated a group of people who had great impact or influence on the project, including some members from PingCAP who founded TiDB. New PMC members are selected based on the rules described above.
+
+For a list of the current PMC members, see the [PMC Members](./pmc-list.md).
+
+<!--- need the link to a Guiding Principals (Missions, Values; to be added separately) page as the principals for PMC -->
+
+### Organizer Committee
+
+The Organizer Committee consists of organizers in charge of community event or activity operations. They ensure the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions.
+
+### Developer Group
+
+As the cornerstone of community development, the TiDB Developer Group consists of these roles: Maintainer, Committer, Reviewer, Active Contributor, and Contributor. Each role takes corresponding responsibilities in the community. Collectively they play an important role in the robust development of TiDB. For more details, refer to [Community Membership](TBD). 
+
+The Developer Group operates TiDB projects in two function forms: Special Interest Group (SIG) and Working Group (WG). The diagram below illustrates the definition of the SIG and WG, the internal roles of the group, and the role promotion path.
```

Comment:
didn't see diagram here

Reply:
I think we need to link to the readme in the architecture.